CHANGELOG.rst

```diff
@@ -58,7 +58,7 @@ New features
 * New flag ``--porcelain`` for programmatic integrations to use. See the
   ``integrations`` section :doc:`here </usage>` for details.
 * Implement a new :doc:`configuration option </configure>`: ``default_due``.
-* The configuration file is now pre-emptively validated. Users will be warned
```

Comment:
Both are right, Oxford is a bit traditionalist: https://www.merriam-webster.com/dictionary/preemptive